### ISSUE TITLE
aws-redshiftserverless: Add Max RPU Support

### ISSUE THREAD

**Issue by IkeNefcy** [aws/aws-cdk#29259]
### Describe the feature

Opening this early. Related: https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1844
Feature likely doesn't already exist due to no CFN support. If anyone wants to help make it a priority for CFN team that would be helpful. (add 👍 to issue above)

### Use Case

Despite there being no support, the CDK deployment still overwrites any manually added max RPU value, which makes this a vital feature for CI/CD.

### Proposed Solution

Once this issue is resolved https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1844,
please add Max RPU support in CDK.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.128.0

### Environment details (OS name and version, etc.)

MacBook Pro M2

**Comment by IkeNefcy** [comment]
Looks like there is no aws-redshiftserverless tag. Bot is confused.

**Comment by tim-finnigan** [comment]
Thanks for the feature request. I'll add the appropriate labels here. Others can add a 👍 to https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1844 to help with visibility.

**Comment by IkeNefcy** [comment]
@tim-finnigan Hey, the issue I mentioned for CFN has been updated. The Max Capacity option has been added. I ran a test with my gamma env and it worked just fine editing the redshift serverless wg capacity. 
Syntax added here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-redshiftserverless-workgroup.html. 
Updating so you know that this CDK ticket is viable to be worked whenever the team gets a chance. Thanks again.
